.github/workflows/main.yml

```diff
@@ -52,7 +76,7 @@ jobs:
       - name: Setup PHP, with composer and extensions
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 7.4
+          php-version: 8.0
```

Comment:
@frederikbosch 

I'd be careful with this change, since there are certain fixers that have - unless configured explicitly - behaviour that is specific to the PHP version that `friendsofphp/php-cs-fixer` is running on. 

Perhaps it would be better to run the `php-cs-fixer` job on the lowest supported version of PHP?

Reply:
I agree with @PowerKiKi here.

Reply:
@localheinz AFAIK our configuration for `friendsofphp/php-cs-fixer` is hand-picked to avoid fixers that are incompatible with the oldest PHP version supported. However, if you are aware of fixer whose behavior changes, then I'd like to know which one, because that would wreak havoc in my others projects too :scream:

Reply:
@PowerKiKi 

That would be all fixers that make use of [`VersionSpecificCodeSample`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/f4b0017d04967bd4b5ad5d74701866e22e1df3c6/src/FixerDefinition/VersionSpecificCodeSample.php), see https://github.com/FriendsOfPHP/PHP-CS-Fixer/search?q=VersionSpecificCodeSample&type=code.

👍  Let's see if we run into problems, nothing to do here for now.